index.html

```diff
@@ -1367,9 +1367,9 @@ <h2>
 ed25519
             </td>
             <td>
-Ed25519 public key values MUST either be encoded as a JWK or be encoded as the
-raw 32-byte public key value in Base58 Bitcoin format using the
-<code>publicKeyBase58</code> property.
+Ed25519 public key values MUST either be encoded as a JWK or be encoded as
+the raw 32-byte public key value in Base58 Bitcoin format using the
+<code>publicKeyBase58</code> property if they are not encoded in JWK format.
```

Comment:
```suggestion
<code>publicKeyBase58</code> property.
```
The trailing clause is now redundant.